app/validators/well_position_validator.rb

```diff
@@ -47,6 +47,12 @@ def validate_contiguousness(record)
     record.errors.add(:wells, "must be in the valid order #{REFERENCE_WELLS}")
   end
 
+  private
+
+  def wells_unless_marked_for_destruction(record)
+    record.wells.filter { |well| !well.marked_for_destruction? }
```

Comment:
Where has this method `marked_for_destruction?` been added? Just can't see it in this PR

Reply:
marked_for_destruction? is a Rails method. When you use the _destroy flag it marks the record for destruction. In this case we need to ensure that any wells that are marked for destruction are not validated here.